cranelift/fuzzgen/src/lib.rs

```diff
@@ -224,14 +273,14 @@ where
         Ok(ctx.func)
     }
 
-    fn generate_func(&mut self) -> Result<Function> {
-        let func = FunctionGenerator::new(&mut self.u, &self.config).generate()?;
+    pub fn generate_func(&mut self, target_triple: Triple) -> Result<Function> {
```

Comment:
I think this doesn't need to be `pub`; it looks like it's only called from this module now.